### PR TITLE
Add support for managing multiple scripts

### PR DESCRIPTION
**Pull request by cbsiddharth**
This patch introduces support for daemonizing multiple scripts resolving #9.

Tested with systemd and initd. Couldn't test on launchd as I don't own a mac. But I presume that should work too.

### CODE REVIEW

File: liquidsoap.systemd.in [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 [Service]
 Type=forking
 User=@user@
-PIDFile=@pid_dir@/run.pid
+PIDFile=@pid_file@
 WorkingDirectory=@base_dir@
 ExecStart=@liquidsoap_binary@ @run_script@
-Restart=always
+#Restart=always
```
Comment (toots) [inline liquidsoap.systemd.in]: Is this intended?

**Comment by toots** [comment]
It looks great, thanks! I'll test & merge soon!

File: liquidsoap.systemd.in (hunk 1 above)
Comment (cbsiddharth) [reply liquidsoap.systemd.in]: Oops. That was when I was debugging something. Will correct, squash and update this branch today.

**Comment by cbsiddharth** [comment]
@toots, Please check now.

**Comment by toots** [comment]
It looks great, thanks. Merging now, will release soon!

**Comment by sandeep318** [comment]
How can I run two liquidsoap instance 

I have two scripts radio1 and radio2 

both are  placed under "liquidsoap-daemon/script"

I was able to start the radio1 using script ./daemonize-liquidsoap.sh radio1

and running fine. 

but when I tried to start another script  radio2 it is giving some error 
++
systemctl status radio2-liquidsoap.service
● radio2-liquidsoap.service - radio2 Liquidsoap daemon
   Loaded: loaded (/etc/systemd/system/radio2-liquidsoap.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
     Docs: http://liquidsoap.fm/

Mar 12 20:14:56 instance-3 systemd[1]: Starting radio2 Liquidsoap daemon...
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: PID file /home/nick/liquidsoap-daemon/pid/radio2-run.pid not readable (yet?) a
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Daemon never wrote its PID file. Failing.
Mar 12 20:14:56 instance-3 systemd[1]: Failed to start radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Unit entered failed state.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Failed with result 'resources'.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Service hold-off time over, scheduling restart.
Mar 12 20:14:56 instance-3 systemd[1]: Stopped radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Start request repeated too quickly.
Mar 12 20:14:56 instance-3 systemd[1]: Failed to start radio2 Liquidsoap daemon.
++

any advice on what might be causing this?

**Comment by toots** [comment]
Have you tried to run this command to see more logs: `sudo journalctl -u radio2-liquidsoap.service`

**Comment by sandeep318** [comment]
Yes, here is the logs 

++
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: PID file /home/nick/liquidsoap-daemon/pid/radio2-run.pid not readable (yet?) a
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Daemon never wrote its PID file. Failing.
Mar 12 20:14:56 instance-3 systemd[1]: Failed to start radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Unit entered failed state.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Failed with result 'resources'.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Service hold-off time over, scheduling restart.
Mar 12 20:14:56 instance-3 systemd[1]: Stopped radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: Starting radio2 Liquidsoap daemon...
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: PID file /home/nick/liquidsoap-daemon/pid/radio2-run.pid not readable (yet?) a
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Daemon never wrote its PID file. Failing.
Mar 12 20:14:56 instance-3 systemd[1]: Failed to start radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Unit entered failed state.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Failed with result 'resources'.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Service hold-off time over, scheduling restart.
Mar 12 20:14:56 instance-3 systemd[1]: Stopped radio2 Liquidsoap daemon.
Mar 12 20:14:56 instance-3 systemd[1]: radio2-liquidsoap.service: Start request repeated too quickly.
Mar 12 20:14:56 instance-3 systemd[1]: Failed to start radio2 Liquidsoap daemon.
++

**Comment by nandababa** [comment]
@toots any ideas why radio2 is failing? @sandeep318 thinks it's not a permission problem 🤓

**Pull request merged**